### PR TITLE
cataclysm-dda: Add persist, rename shortcuts

### PR DESCRIPTION
**Pull request by satorope**
Autoupdate now works.
Add persist data.
Rename shortcuts according to official site.

### CODE REVIEW

File: bucket/cataclysm-dda.json [hunk 1]
```diff
@@ -1,8 +1,11 @@
 {
     "homepage": "https://cataclysmdda.org",
-    "description": "Turn-based survival game set in a post-apocalyptic world (with text-based graphics)",
+    "description": "Roguelike in a post-apocalyptic world (with text-based graphics)",
     "version": "0.D",
-    "license": "CC-BY-SA-3.0",
+    "license": {
+        "identifier": "CC-BY-SA-3.0,GPL-2.0+,OFL-1.0,BSL-1.0,Zlib,MIT,BSD(libbacktrace)",
+        "url": "https://github.com/CleverRaven/Cataclysm-DDA/blob/master/LICENSE.txt"
+    },
```
Comment (Calinou) [inline bucket/cataclysm-dda.json:8]:
```suggestion
    "license": "CC-BY-SA-3.0",
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Comment by satorope** [comment]
CC-BY-SA-3.0 is a license for Cataclysm, which is the basis of CDDA, and I suspect that fonts and other things are not included in that license.
The license text looks the same.
https://github.com/Whales/Cataclysm/blob/master/LICENSE

The creator should define the Cataclysm-dda license, but it seems to be neglecting that and listing the licenses for the materials used.
CC-BY-SA-3.0 is not possible when GPL etc. are incorporated.
In particular, fonts are only attached, so license restrictions will be inherited when redistributed.

I also mistook the OFL license.
OFL-1.1 is correct.

**Comment by satorope** [comment]
If it is a rule to write only the license of the main program itself, I will leave the decision to you because the above story may be inappropriate.

**Pull request merged**